### PR TITLE
Removed some packages and installed them as dev dependencies

### PR DESCRIPTION
**Pull request by sibshahz**
@gbowne1 

Removed the dependencies mentioned in this [issue](https://github.com/gbowne1/reactsocialnetwork/issues/196) and installed them as Dev Dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,78 +1,79 @@
...
+ 	 	"babel-eslint": "^10.1.0",
+ 	 	"babel-loader": "^9.1.2",
+ 	 	"eslint-config-prettier": "^8.8.0",
+ 	 	"eslint-config-react-app": "^7.0.1",
+ 	 	"eslint-plugin-prettier": "^4.2.1",
+ 	 	"eslint-plugin-react": "^7.32.2",
+ 	 	"jest-watch-typeahead": "^0.6.5",
+ 	 	"prettier": "^2.8.8",
+ 	 	"react-test-renderer": "^18.2.0",
+ 	 	"weback": "^0.0.1-security"
```
Comment (pawel975) [inline package.json:77]: I think you misspeled 'webpack', in previous version webpack is installed and after update isn't.

Comment (gbowne1) [reply package.json:77]: Thats a easy mistake to make.  I also don't think we need webpack unless we really need a config for webpack thats outside of the one built in create react app.  

We do need to minify the bundle.  Its currentky not doing that if you look at console.

Comment (sibshahz) [reply package.json:77]: Yes I did, gonna make it right.

**Review by sibshahz — COMMENTED** [review]
@gbowne1 resubmitting after fixing weback package to webpack.

**Comment by gbowne1** [comment]
This still has conflicts with the json files.

**Comment by sibshahz** [comment]
Conflicts removed.

**Review by pawel975 — APPROVED** [review]

**Review by gbowne1 — APPROVED** [review]
Approving these changes for merge

**Review by manuel12 — APPROVED** [review]
Approving changes!

**Pull request merged**